bciers/apps/compliance/src/tests/components/compliance-summaries/ComplianceSummariesPage.test.tsx

```diff
@@ -82,4 +93,38 @@ describe("ComplianceSummariesPage", () => {
     ).toBeVisible();
     expect(screen.getByText(/due on November 30, 2025/i)).toBeVisible();
   });
+
+  it("renders alert message with 'was' if current date is past due", async () => {
```

Comment:
I think this one needs custom mock data that has all obligations met, or else it's testing the same situation as `renders alert message for overdue penalty if date is past due and initial data has obligation not met`.

Relatedly, I'd suggest that all of the tests assert both alerts (that they're either there or not there) to make sure all the combinations are covered.